src/js/constants/event-names.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
think it would be better to have one file: `events.js` and create an object that refers to something like: 
```
events: {
   eventBus: { ... }
   gtm: { ... }
}